test/config.js

```diff
@@ -21,46 +21,49 @@ var config = require('../lib/config')
 var Instrumentation = require('../lib/instrumentation')
 
 process.env.ELASTIC_APM_METRICS_INTERVAL = '0'
+process.env.ELASTIC_APM_CENTRAL_CONFIG = 'false'
 
 var optionFixtures = [
-  ['serviceName', 'SERVICE_NAME', 'elastic-apm-node'],
-  ['secretToken', 'SECRET_TOKEN'],
-  ['serverUrl', 'SERVER_URL'],
-  ['verifyServerCert', 'VERIFY_SERVER_CERT', true],
-  ['serviceVersion', 'SERVICE_VERSION'],
+  ['abortedErrorThreshold', 'ABORTED_ERROR_THRESHOLD', 25],
   ['active', 'ACTIVE', true],
-  ['logLevel', 'LOG_LEVEL', 'info'],
-  ['hostname', 'HOSTNAME'],
   ['apiRequestSize', 'API_REQUEST_SIZE', 768 * 1024],
   ['apiRequestTime', 'API_REQUEST_TIME', 10],
-  ['frameworkName', 'FRAMEWORK_NAME'],
-  ['frameworkVersion', 'FRAMEWORK_VERSION'],
-  ['stackTraceLimit', 'STACK_TRACE_LIMIT', 50],
-  ['captureExceptions', 'CAPTURE_EXCEPTIONS', true],
-  ['filterHttpHeaders', 'FILTER_HTTP_HEADERS', true],
+  ['asyncHooks', 'ASYNC_HOOKS', true],
+  ['captureBody', 'CAPTURE_BODY', 'off'],
   ['captureErrorLogStackTraces', 'CAPTURE_ERROR_LOG_STACK_TRACES', config.CAPTURE_ERROR_LOG_STACK_TRACES_MESSAGES],
+  ['captureExceptions', 'CAPTURE_EXCEPTIONS', true],
   ['captureSpanStackTraces', 'CAPTURE_SPAN_STACK_TRACES', true],
-  ['captureBody', 'CAPTURE_BODY', 'off'],
+  ['centralConfig', 'CENTRAL_CONFIG', true],
```

Comment:
This is one of two lines that I added (the diff is a little hard to read as I reordered the array to be alphabetically sorted)